### PR TITLE
Full version branch

### PR DESCRIPTION
**Pull request by Theodore354**
Okay so in this branch i made changes to the project as requested my microverse. I added the work section and implemented the css grid i learned in the curriculum. I followed the github workflow and did all that was required. I also used the ul and li tags as requested to make my work look more professional

### CODE REVIEW

**Review by Gambit142 — COMMENTED** [review]
Hi @Theodore354 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have a pull request summary, kindly describe what you did in this branch as the pull request summary

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by sheylaPozo** [comment]
# Required Changes ♻️ 
Hi dear student @Theodore354, 

**Good job so far!**
You still need to work on some issues to go to the next project, but you are almost there!

Good points 👍
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your commit messages are descriptive ✔️

**Aspects to improve ♻️**
 Check the comments under the review.
**Optional suggestions**
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by sheylaPozo** [comment]
# ♻ Required changes ♻

- While the project is almost complete, you still have some changes yet to do to get approved, as it is a mobile version, it has to match with the original Figma template, in your case template 3, in all the aspects (images in backgrounds included, and dimensions)

**Example :** 

Figma version        | Your Mobile version
-------------------- | ----------------------
template |  Mobile

if we compare both versions, the Figma version at the 100% percent shows the icons of the social media, while yours in 100% doesn't show them, and this is due the dimensions you used on your CSS,  kindly change the dimensions to make it fit and look like in the template example, also you're missing one of the background images in the mobile version... 
### Recommendations :
temp Kindly make sure to always check to have all the background images in your project when you go into the mobile version, I see you put the image there on the desktop size but we aren't reviewing desktop this time, only mobile, and it should be included there in your project.  

- [ ] If you have any doubts or questions make sure to leave them under the comment section..

sticker

**Comment by sheylaPozo** [comment]
# ♻ Required changes ♻

- While the project is almost complete, you still have some changes yet to do to get approved, as it is a mobile version, it has to match with the original Figma template, in your case template 3, in all the aspects (images in backgrounds included, and dimensions)

**Example :** 

Figma version        | Your Mobile version
-------------------- | ----------------------
template |  Mobile

if we compare both versions, the Figma version at the 100% percent shows the icons of the social media, while yours in 100% doesn't show them, and this is due to the dimensions you used on your CSS,  kindly change the dimensions to make it fit and look like in the template example, also you're missing one of the background images in the mobile version... 
### Recommendations :
temp Kindly make sure to always check to have all the background images in your project when you go into the mobile version, I see you put the image there on the desktop size but we aren't reviewing desktop this time, only mobile, and it should be included there in your project.  

- [ ] If you have any doubts or questions make sure to leave them under the comment section..


sticker

**Comment by sheylaPozo** [comment]
# ♻ Required changes ♻

- While the project is almost complete, you still have some changes yet to do to get approved, as it is a mobile version, it has to match with the original Figma template, in your case template 3, in all the aspects (images in backgrounds included, and dimensions)

### **Example :** 

Figma version        | Your Mobile version
-------------------- | ----------------------
template |  Mobile

if we compare both versions, the Figma version at the 100% percent shows the icons of the social media, while yours in 100% doesn't show them, and this is due to the dimensions you used on your CSS,  kindly change the dimensions to make it fit and look like in the template example, also you're missing one of the background images in the mobile version... 
### Recommendations :

temp 

- Kindly make sure to always check to have all the background images in your project when you go into the mobile version, I see you put the image there on the desktop size but we aren't reviewing desktop this time, only mobile, and it should be included there in your project.  
---

sticker

---
- [ ] If you have any doubts or questions make sure to leave them under the comment section..

File: index.html [hunk 1]
```diff
@@ -66,6 +66,178 @@ <h2 class="heading2">I am a Software Developer</h2>
           </a>
         </li>
       </ul>
```
Comment (sheylaPozo) [inline index.html:68]: # ♻ Required changes ♻

- While the project is almost complete, you still have some changes yet to do to get approved, as it is a mobile version, it has to match with the original Figma template, in your case template 3, in all the aspects (images in backgrounds included, and dimensions)

### **Example :** 

Figma version        | Your Mobile version
-------------------- | ----------------------
template |  Mobile

if we compare both versions, the Figma version at the 100% percent shows the icons of the social media, while yours in 100% doesn't show them, and this is due to the dimensions you used on your CSS,  kindly change the dimensions to make it fit and look like in the template example, also you're missing one of the background images in the mobile version... 
### Recommendations :

temp 

- Kindly make sure to always check to have all the background images in your project when you go into the mobile version, I see you put the image there on the desktop size but we aren't reviewing desktop this time, only mobile, and it should be included there in your project.  
---

sticker

---
- [ ] If you have any doubts or questions make sure to leave them under the comment section..

**Review by sheylaPozo — CHANGES_REQUESTED** [review]

**Comment by sheylaPozo** [comment]
###  ♻ Required changes ♻
readme
images.githubusercontent.com/54015740/143613927-96b7d6fe-a2db-452a-b1da-fa4266fdcaf0.png)

- Remember for a professional-looking readme file Microverse's request is to have all the links on the readme file be functional and direct us to the right page, in this case, your Mit license isn't linked to a valid link, I won't invalidate your project for that for this time since your project was already invalidated before but I kindly suggest to replace that link or remove it completely from the license part, also is all about presentation and there's an extra icon in the Authors part, that would look better if it was removed.
`readme`
 ----

**Comment by sheylaPozo** [comment]
## Optional Suggestions 
---

- [x]  Kindly try to be more descriptive on the PR for the reviewers to know what changes you made and what you worked on, always remember that's part of the requirements for presentation, also make sure to be more kind, polite and respectful in the  description of your PR, is a valid reason for project invalidation, I see that you have tried to make the effort to describe it and I understand not all know how to properly describe the project in a manner for everyone to understand the changes clearly and for it to look more proffesional  👍 💯 

Your PR description       |  Example of descriptive PR
-------------------------- | -----------------------------
optional
om/54015740/127129480-dfb12898-5429-4ff7-a67a-f4e3df0edc64.gif) | descriptive

![tenor](https://user-images.githubusercontent.

**Comment by sheylaPozo** [comment]
## Optional Suggestions 📌📌
---

- [ ] Kindly try to be more descriptive on the PR for the reviewers to know what changes you made and what you worked on, always remember that's part of the requirements for presentation, also make sure to be more kind, polite, and respectful in the description of your PR, is a valid reason for project invalidation, I see that you have tried to make the effort to describe it and I understand not all know how to properly describe the project in a manner for everyone to understand the changes clearly and for it to look more professional  👍 💯 

Your PR description       |  Example of descriptive PR
-------------------------- | -----------------------------
optional
om/54015740/127129480-dfb12898-5429-4ff7-a67a-f4e3df0edc64.gif) | descriptive

tenor
---
That's all :) keep doing a good job, and happy coding 💪😎💯

**Comment by sheylaPozo** [comment]
## Optional Suggestions 📌📌
---

- [ ] Kindly try to be more descriptive on the PR for the reviewers to know what changes you made and what you worked on, always remember that's part of the requirements for presentation, also make sure to be more kind, polite, and respectful in the description of your PR, is a valid reason for project invalidation, I see that you have tried to make the effort to describe it and I understand not all know how to properly describe the project in a manner for everyone to understand the changes clearly and for it to look more professional  👍 💯 

Your PR description       |  Example of descriptive PR
-------------------------- | -----------------------------
optional
om/54015740/127129480-dfb12898-5429-4ff7-a67a-f4e3df0edc64.gif) | descriptive

---

digilah-digifun

`That's all :) keep doing a good job, and happy coding 💪😎💯`

**Comment by sheylaPozo** [comment]
## Optional Suggestions 📌📌
---

- [ ] Kindly try to be more descriptive on the PR for the reviewers to know what changes you made and what you worked on, always remember that's part of the requirements for presentation, also make sure to be more kind, polite, and respectful in the description of your PR, is a valid reason for project invalidation, I see that you have tried to make the effort to describe it and I understand not all know how to properly describe the project in a manner for everyone to understand the changes clearly and for it to look more professional  👍 💯 

Your PR description       |  Example of descriptive PR
-------------------------- | -----------------------------
optional| descriptive

---

digilah-digifun

`That's all :) keep doing a good job, and happy coding 💪😎💯`

**Comment by sheylaPozo** [comment]
## Optional Suggestions 
---

- [ ] As best practices remember to add all the images into the "img" folder because it makes it look more professional.

folder
coding
---
###  Happy coding 🎉😄

File: index.html [hunk 2]
```diff
@@ -66,6 +66,156 @@ <h2 class="heading2">I am a Software Developer</h2>
...
+      </h1>
+      <p class="about-me-description center-textx">
+        I can help you build a product , feature or website Look through some of
+        my work and experience! If you like what you see and have a project you
+        need coded, don’t hesitated to contact me.
+      </p>
+      <span class="self-center">
+        <a href="#" class="action-btn">Get My Resume</a>
+      </span>
+    </section>
```
Comment (andersonlebon) [inline index.html:171]: ## [OPTIONAL]
- [ ] Kindly work on the `background` of the about section, this should have the same `background` as the template, Check dow here :point_down:

Your design | the template
---------------- | -----------------
Screenshot from 2021-11-27 18-55-57 | Screenshot from 2021-11-27 19-04-42
 
### kindly make this change as **IMPORTANT** one

**Review by andersonlebon — APPROVED** [review]
### Hi @Theodore354 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- You  are following GitHub actions :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is now professional :heavy_check_mark: 

## Optional suggestions
- [ ] Kindly Check the comments under the review._
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 2 above)
Comment (Theodore354) [reply index.html:171]: okay

**Pull request merged**